cudapoa/src/cudapoa_nw_banded.cuh

```diff
@@ -556,6 +556,122 @@ __device__ __forceinline__
     return aligned_nodes;
 }
 
+// global kernel used in testing, hence uses int16_t for SizeT and ScoreT,
+// may need to change if test inputs change to long reads
+template <typename SizeT>
```

Comment:
removed `SizeT` instances in tests files, replaced with `int16_t`. But since `runNW` is defined in `cudapoa_nw.cuh`, to avoid linking errors, it's being kept templetized.